backend/src/StdLibExecution/Libs/List.fs

```diff
@@ -1365,6 +1348,7 @@ let fns : List<BuiltInFn> =
       previewable = Pure
       deprecated = NotDeprecated }
 
+    // CLEANUP: still used in listPackageFilesOnDisk and LoadPackagesIntoDarkCanvas
```

Comment:
Yes, when I add the `PACKAGE.Darklang.Stdlib.` to the `List.iter` calls I get function not found

Reply:
Ah, I see - I'm guessing there's an issue with NameResolution in WT2PT that prevents it getting resolved.

Those 2 lines _start_ with the List.iter, while the other ones are in pipes. We probably handle those differently somehow in WT2PT/NameResolution.

Worth looking around there to see if you can find what's missing. I suspect these areas might be a bit unclear though, so free to pair if useful :)

This might not be the cause but it's a strong hunch